charts/dvpe-deployment-gloo/values.yaml

```diff
@@ -194,38 +194,68 @@ gloo:
+              # gloo.authConfig.spec.configs.tokenValidationPlugin.config.cache.awsRegion -- `awsRegion` where the cache is located
+              awsRegion: eu-west-1
+            # gloo.authConfig.spec.configs.tokenValidationPlugin.config.oidcUrl -- `oidcUrl` where the access token can be verified at the IDP
             oidcUrl:
-            # gloo.authConfig.spec.configs.backendPlugin.config.jwksUrl -- `jwksUrl` where the JWKS (JSON Web Key Store) can be retrieved from the IDP
-            jwksUrl:
-        m2mPlugin:
-          # gloo.authConfig.spec.configs.m2mPlugin.enabled -- If `enabled` set to true the machine to machine plugin will be used
+            # gloo.authConfig.spec.configs.tokenValidationPlugin.config.allowedClientIds -- `allowedClientIds` **list (NOT string!)** of ids that are allowed by the plugin. If not given at all, all clients are allowed. If [], then no client is allowed. If [a, b], then a, b are allowed
+            allowedClientIds:
+            # gloo.authConfig.spec.configs.tokenValidationPlugin.config.authLevel -- `authLevel` strong authentication level: nil/4000/7000
+            authLevel:
+        clientCredentialsPlugin:
+          # gloo.authConfig.spec.configs.clientCredentialsPlugin.enabled -- If `enabled` set to true the machine to machine plugin will be used
           enabled: false
-          # gloo.authConfig.spec.configs.m2mPlugin.name -- `Name` of the cache plugin
-          name: AuthM2m
+          # gloo.authConfig.spec.configs.clientCredentialsPlugin.name -- `Name` of the auth client credentials plugin
+          name: AuthClientCredentials
           config:
-            # gloo.authConfig.spec.configs.m2mPlugin.config.cacheTableName -- `cacheTableName` in DynamoDB of the auth cache
-            cacheTableName: auth-cache-prod
-            # gloo.authConfig.spec.configs.m2mPlugin.config.awsRegion -- `awsRegion` where the cache is located
-            awsRegion: eu-west-1
-            # gloo.authConfig.spec.configs.m2mPlugin.config.amBaseUrl -- `amBaseUrl` (access management base URL) - where the access token can be verified at the IDP
-            amBaseUrl:
-            # gloo.authConfig.spec.configs.m2mPlugin.config.clientId -- `clientId` of the machine2machine client registered at the IDP
+            # `cache` of the auth cache
+            cache:
+              # gloo.authConfig.spec.configs.clientCredentialsPlugin.config.cache.enabled -- if `enabled` is false, no cache is used
+              enabled: false
+              # gloo.authConfig.spec.configs.clientCredentialsPlugin.config.cache.tableName -- `tableName` of the auth cache
+              tableName:
+              # gloo.authConfig.spec.configs.clientCredentialsPlugin.config.cache.awsRegion -- `awsRegion` where the cache is located
+              awsRegion:
+            # gloo.authConfig.spec.configs.clientCredentialsPlugin.config.oidcUrl -- `oidcUrl`  - where the access token can be verified at the IDP
+            oidcUrl:
```

Comment:
`oidcUrl` is in this case not correct. Maybe we should name it `idpUrl` or ´identityProviderUrl`

Reply:
done

Reply:
I had a discussion with Alex and WebEAM provides all m2m specific configurations with the .wellknown/openid-configuration endpoint. We agreed that we should name this variable `oidcUrl` like in the plugins. 
Sorry for the extra work.

Reply:
I changed it back after consulting with Alex - there was a similar discussion in the other PR